src/Components/Patient/FileUpload.tsx

```diff
@@ -159,6 +198,9 @@ export const FileUpload = (props: FileUploadProps) => {
 
   const handleClose = () => {
     setDownloadURL("");
+    setZoom(3);
```

Comment:
You are triggering multiple state updates here and it's guaranteed that these won't occur synchronously.

When we have multiple state updates we would ideally make the state an object and update it.

```
setState({...state, zoom: 3, downloadURL: "", zoomOutDisabled: false, zoomInDisabled: false})
```

Reply:
Each of the state has its own function to compute its value. If we combine them it will take a lot of time for state to update. Also since each of these occur async so we don't know which state will be picked up by  which setState

Reply:
Changes done @bodhish